RFCs/0016-Loggers.md

```diff
@@ -0,0 +1,85 @@
+# 0016 Loggers
```

Comment:
Request to make the title and doc name little more elaborate. This may help understand the intent from looking at filename.

Reply:
Sure. Will elaborate the title.